### PR TITLE
[MC-2302] Update HTML template for inapp preview.

### PR DESCRIPTION
**Pull request by nishant-clevertap**
- Updated HTML template for inapp preview.
- Added check for inapp advanced builder type also.

### CODE REVIEW

**Comment by nzagorchev** [comment]
@nishant-clevertap I see some differences when comparing the HTML with the one on Android, can you please check? https://github.com/CleverTap/clevertap-android-sdk/pull/680

**Review by akashvercetti — APPROVED** [review]

File: CleverTapSDK/InApps/resources/image_interstitial.html
Comment (nishant-clevertap) [inline CleverTapSDK/InApps/resources/image_interstitial.html:1]: @nzagorchev The html file is same as in Android develop branch, I have checked the file difference. Can you also confirm if you are talking about this one only: https://github.com/CleverTap/clevertap-android-sdk/blob/develop/clevertap-core/src/main/assets/image_interstitial.html

Comment (nzagorchev) [reply CleverTapSDK/InApps/resources/image_interstitial.html:1]: yes, resolved.

**Review by akashvercetti — APPROVED** [review]
looks good 👍

**Pull request merged**